### PR TITLE
Update docs-page-update.md

### PR DESCRIPTION
**Pull request by DFriend01**
Related to #4. Updating docs update page.

### CODE REVIEW

**Review by patrick-5546 — COMMENTED** [review]
Do we want section descriptions be rendered or not? If not, we can convert them to markdown comments.

```markdown
<!-- this is a markdown comment -->
```

**Comment by DFriend01** [comment]
Prompts are commented out.

File: .github/ISSUE_TEMPLATE/docs-page-update.md [hunk 1]
```diff
@@ -6,12 +6,15 @@ labels: documentation, update-page
 assignees: ''
 
 ---
-
 ### Purpose
 
+[//]: # (Why is this issue being created? What is hoped to be achieved by addressing this issue?)
```
Comment (patrick-5546) [inline .github/ISSUE_TEMPLATE/docs-page-update.md:11]: Can be more specific: why does this page need to be updated?

File: .github/ISSUE_TEMPLATE/docs-page-update.md [hunk 2]
```diff
@@ -6,12 +6,15 @@ labels: documentation, update-page
...
 
 ### Changes
 
+[//]: # (What changes will be made to the existing docs page and why?)
 
 ### Resources
+
+[//]: # (Link to any extra resources that could help with updating the docs page.)
```
Comment (patrick-5546) [inline .github/ISSUE_TEMPLATE/docs-page-update.md:19]: See https://github.com/UBCSailbot/.github/pull/6#discussion_r1010006890

**Review by patrick-5546 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**